book/src/guest-libs/p256.md

```diff
@@ -0,0 +1,25 @@
+# P256
+
+The P256 guest library uses [`openvm-ecc-guest`](../custom-extensions/ecc.md) to provide elliptic curve operations over the Secp256r1 curve.  
```

Comment:
mention this is meant to be a patch of the `p256` crate and can be swapped in for signature verification usage. Signing from private key is not supported